doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -59,13 +59,31 @@
 <a name="3.2.2"></a>
 ### 3.2.2 Container Networking Services
 
-> This chapter should discuss networking related considerations of container network services, like:
-> * Ingress and any limitations on protocol or raw interface support
-> * Dependencies on external loadbalancers
-> * Regular service mesh related considerations
-> * Network Service Mesh related considerations
-> * Number of interfaces and IP addresses to be supported by a pod
-> * IPv6 single and dual stack needs
+Becouse `req.inf.ntw.01` requires the architecture to support CNI and `req.inf.ntw.16` requires the capability to attach several network interfaces to the pods the architecture must suport a CNI metaplugin/CNI multiplexer.
+
+A CNI metaplugin/CNI multiplexer is capable to attach several interfaces, using different other CNI plugins to a pod. The different network charastheristics of the interfaces, which require different networking technologies require different CNI plugins.
+
+Inter node communication required by `req.inf.ntw.02` must be served by a simple CNI using overlay technology, like Flannel.
+
+There are two types of low latency and high throughput networks required by `req.inf.ntw.04`. Network used for signalling traffic are more demanding than what an overlay network can handle, but still does not need the usage of user space networking. Due to the nature of signalling protocols used this type of netorks require the NAT-less communication stated by `req.inf.ntw.03`. Due to the combination of these two requirements networks with this charactheristics must be served by a CNI plugin with IPVLAN or MACVLAN suport.
```

Comment:
"Network used for signalling traffic are more demanding than what an overlay network can handle" - I don't disagree, but we should call out the reasons why, I think.
typo netorks > networks
typo charactheristics > characteristics
typo suport > support
"must be served by a CNI plugin with IPVLAN or MACVLAN suport" - should we describe why IPVLAN and MACVLAN support? What is it about them that makes them work? What are the alternatives that aren't appropriate?

Reply:
See above: need not be served by anything involving the CNI, for one, and macvlan support is certainly not a requirement for control communications.  Iimagine a BGP RR that insisted on this specific requirement.  Now imagine it if it was using bonded uplinks...

Reply:
@tomkivlin the reason is the throughput need and the need for NAT less communication. 

@iawells from what is currently available today I see no other solutions than CNI based ones, but maybe I miss something.